src/main.rs

```diff
@@ -31,25 +31,16 @@ extern crate rand;
 // --
 
 pub use samp_udp_proxy::config::Config;
-pub use samp_udp_proxy::server::Server;
 pub use samp_udp_proxy::proxy::Proxy;
+pub use samp_udp_proxy::server::Server;
 
 // --
 //  Main
 // --
 
-pub fn main() {
-    let config: String = "config.toml".to_string();
-
-    // Parse Config
-    let config = Config::parse(config).unwrap();
+const CONFIG: &str = "config.toml";
```

Comment:
This would be better as an environment variable interface. It would also be nice to build a simple CLI too, check out https://github.com/clap-rs/clap